docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,390 +1,552 @@
+| `StartPosition` | `Vector2`                        | For linear only | The start position of the linear gradient - coordinate system depends on the `Units` value. |
+| `EndPosition`   | `Vector2`                        | For linear only | The end position of the linear gradient - coordinate system depends on the `Units` value.   |
+| `Center`        | `Vector2`                        | For radial only | The center point of the radial gradient - coordinate system depends on the `Units` value.   |
+| `Radius`        | `float`                          | For radial only | The size of the radial gradient radius - coordinate system depends on the `Units` value.    |
+| `StopColor`     | `PropertyArray` of `Color`       | Yes             | The color at the stop offsets. At least two are required to show a gradient.                |
+| `StopOffset`    | `PropertyArray` of `floats`      | No              | The stop offsets in relative units (default is 0.0 and 1.0) - to see all colors from the `StopColor` the lengths of these two arrays should be the same.            |
+| `Units`         | `GradientVisualUnitsType`        | No              | Defines the coordinate system for the attributes: start and end points for a linear gradient, center point and radius for a radial gradient (see tab. [with possible values](#table_grad_possible_values)). |
+| `SpreadMethod`  | `GradientVisualSpreadMethodType` | No              | Indicates what happens if a gradient starts or ends inside bounds (default val. - `Pad`; see tab. [with possible values](#table_grad_possible_values)).            |
+
+
+<a name="table_grad_possible_values"></a>
+**Table: Possible values of chosen types used by the GradientVisual properties**
+
+| Type                             | Possible value      | Description                                                                                                  |
+|----------------------------------|---------------------|--------------------------------------------------------------------------------------------------------------|
+| `GradientVisualUnitsType     `   | `ObjectBoundingBox` | The coordinate system in which the top-left is (-0.5, -0.5) and the bottom-right is (0.5, 0.5) - default.    |
+|                                  | `UserSpace`         | The system with the top-left being (0, 0) and the bottom-right - (control width, control height).            |
+| `GradientVisualSpreadMethodType` | `Pad`               | Uses the terminal colors of the gradient to fill the remainder of the area - default.                        |
+|                                  | `Reflect`           | Reflects the gradient pattern start-to-end, end-to-start, start-to-end, and so on, until the area is filled. |
+|                                  | `Repeat`            | Repeats the gradient pattern start-to-end, start-to-end, start-to-end, and so on, until the area is filled.  |
+
+
 **Usage:**
```

Comment:
I suggest te following heading here: 
#### Use GradientVisuals

What is your opinion?